packages/skill-lesson/hooks/use-global-player-shortcut.ts

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import {MuxPlayerRefAttributes} from '@mux/mux-player-react/*'
+
+const ignoredInputs = ['input', 'select', 'button', 'textarea', 'mux-player']
+
+export const useGlobalPlayerShortcuts = (muxPlayerRef: {
+  current: MuxPlayerRefAttributes
+}) => {
+  const handleUserKeyPress = React.useCallback((e: any) => {
+    const activeElement = document.activeElement
+    if (
+      activeElement &&
+      ignoredInputs.indexOf(activeElement.tagName.toLowerCase()) === -1
+    ) {
+      if (muxPlayerRef.current) {
+        if (e.key === ' ') {
+          e.preventDefault()
+          muxPlayerRef.current.paused
+            ? muxPlayerRef.current.play()
+            : muxPlayerRef.current.pause()
+        }
+        if (e.key === 'ArrowRight') {
+          e.preventDefault()
+          muxPlayerRef.current.currentTime =
+            muxPlayerRef.current.currentTime +
+            (muxPlayerRef.current.forwardSeekOffset || 10)
+        }
+        if (e.key === 'ArrowLeft') {
+          e.preventDefault()
+          muxPlayerRef.current.currentTime =
+            muxPlayerRef.current.currentTime -
+            (muxPlayerRef.current.forwardSeekOffset || 10)
+        }
```

Comment:
added these three for now, but we could have `f` for fullscreen and perhaps more?

Reply:
YT is the muse here